src/FrontPage.jsx

```diff
@@ -0,0 +1,34 @@
+import { useNavigate } from "react-router-dom";
+
+function FrontPage(){ 
+  const navigate = useNavigate();
+  function handleClick(path){
```

Comment:
Måske bruge et andet funktionsnavn der bedre reflektere hvad funktionen gør :)
```suggestion
  function navigateTo(path){
```